spritecontrol.cpp

```diff
@@ -332,3 +337,50 @@ std::map<PatternName, GlobalPlayer::MoveFunction> GlobalPlayer::move_functions {
 		}
 	}},
 };
+
+std::vector<PatternName> PatternRepository::load_disabled_patterns() {
+	Registry registry;
+
+	std::vector<PatternName> disabled_patterns = { };
+
+	std::vector<std::wstring> disabled_patterns_strings = split<std::wstring>(registry.get_string(disabled_patterns_name, disabled_patterns_default), disabled_patterns_string_delimiter);
```

Comment:
This shit is so noisy and hard to read but it's just C++ being C++, I have no suggestions for doing it differently and would have done it the same :(